tests/wpt/meta/resize-observer/observe.html.ini

```diff
@@ -0,0 +1,4 @@
+[observe.html]
+  expected: ERROR
```

Comment:
Do we know what the error is here?

Reply:
Yes, it is a missing API. See https://github.com/servo/servo/issues/31182